lib/fob/ordering.ex

```diff
@@ -82,10 +82,14 @@ defmodule Fob.Ordering do
           expr: {:%{}, _, [{:|, _, [{:&, _, [0]}, merges]}]}
         }
       }) do
-    Map.new(merges, fn {toplevel_name,
-                        {{:., _, [{:&, _, [table]}, name_in_table]}, _, _}} ->
-      {{table, name_in_table}, toplevel_name}
+    Map.new(merges, fn
+      {toplevel_name, {{:., _, [{:&, _, [table]}, name_in_table]}, _, _}} ->
+        {{table, name_in_table}, toplevel_name}
+
+      {_toplevel_name, _ast_expr} = expr ->
+        {:merged_computed_column, expr}
     end)
+    |> Map.delete(:merged_computed_column)
```

Comment:
I'm guessing there is a nicer way to build this Map, while also rejecting some values.

The key idea is that we don't need to have all selects in the select merge, if I fully understand the keystoned queries. This filters out queries that are more that just a field/column access.

Reply:
I think [`:maps.filtermap/2`](https://www.erlang.org/doc/man/maps.html#filtermap-2) or an `Enum.reduce/3` would be a little clearer here from a readability perspective